packages/v4/patternfly-docs/content/get-started/design.md

```diff
@@ -108,38 +137,6 @@ Currently, all PatternFly text styles use Red Hat Text and Red Hat Display fonts
 
 When inserting text into your design, always assign it a text style by selecting the text item, followed by selecting a text style from the appearance menu on the right toolbar.
 
-## Designing with PatternFly
-PatternFly offers a variety of resources to help you make the most of the design system, including design guidelines for style, content, and component usage and behavior. You can use these assets in combination with the design kit to create consistent and usable product experiences.
-
-### Style guidelines
-What typography should you be using? What about colors? Spacing? Check out styles to learn everything you need to know about our foundational design elements.
-
-[**View styles**](/guidelines/colors) <i class="ws-content-blueArrow fas fa-arrow-right pf-u-mx-sm"></i>
-
-### Get familiar with usage and behavior guidelines.
-Should you use a wizard or a form to solve this problem? How do you write a good button label? Which navigation makes the most sense? Check out our design guidelines to choose and use the right components for your use case.
-
-[**Usage and behavior guidelines**](/components/about-modal/design-guidelines) <i class="ws-content-blueArrow fas fa-arrow-right pf-u-mx-sm"></i>
-
-### Content guidelines
-How do you write great microcopy and text for your user interface? Read our content guidelines for general principles, guidance on voice, and a style guide to help you with everything from capitalization to date and time formats.
-View content guidelines
-
-[**View content guidelines**](/ux-writing/about) <i class="ws-content-blueArrow fas fa-arrow-right pf-u-mx-sm"></i>
-
-
-
-### Start building with components.
-What does a toolbar look like? What about a filter chip? Components are the building blocks of the design system. Check out individual component pages to view examples, understand states and behaviors, and check out the code samples (if you’re into that kind of thing).
-
-[**View components**](/components/about-modal) <i class="ws-content-blueArrow fas fa-arrow-right pf-u-mx-sm"></i>
-
-### Still feeling unsure?
-See all the pieces working together with demos.
-
-[**View demos**](/demos/primary-detail) <i class="ws-content-blueArrow fas fa-arrow-right pf-u-mx-sm"></i>
-
-
 ## Staying up to date
 Adjustments and bug fixes are made to the kit on a regular basis. When we make a change to any of the libraries, you will be notified via a purple pop-up in the top right corner of the Sketch window that says “Shared Library Update Available”. To update, simply click on the pop-up and followed by “download” to update your library.
```

Comment:
Just realized this has changed a little. It should say that you'll be notified by a bell on the top right corner of your sketch window, then to click on it, and click "Library update available". A pop-up will appear, and you should click "update" for the PF library.

Feel free to work that in a way that makes more sense hahah